source/Background/Controller.js

```diff
@@ -29,6 +29,32 @@ const backgroundController = new BackgroundController({
 
 backgroundController.start();
 
+const fetchCanistersInfo = async (whitelist) => {
+  if (whitelist && whitelist.length > 0) {
+    const canistersInfo = await Promise.all(whitelist.map(async (id) => {
+      const canisterInfo = await PlugController.getCanisterInfo(id);
+
+      if (canisterInfo) {
+        return {
+          id,
+          ...canisterInfo,
+        };
+      }
+
+      return { id };
+    }));
```

Comment:
we should add a try catch in case dab fails

Reply:
good catch, will do before release